internal/impl/kafka/enterprise/redpanda_migrator_offsets_input.go

```diff
@@ -139,23 +123,20 @@ func init() {
 				}
 			}
 
-			if d.RackID, err = conf.FieldString(rmoiFieldRackID); err != nil {
+			var rackID string
+			if rackID, err = conf.FieldString(rmoiFieldRackID); err != nil {
 				return nil, err
 			}
+			clientOpts = append(clientOpts, kgo.Rack(rackID))
 
-			if d.FetchMaxBytes, err = kafka.BytesFromStrFieldAsInt32(rmoiFieldFetchMaxBytes, conf); err != nil {
-				return nil, err
-			}
-			if d.FetchMinBytes, err = kafka.BytesFromStrFieldAsInt32(rmoiFieldFetchMinBytes, conf); err != nil {
-				return nil, err
-			}
-			if d.FetchMaxPartitionBytes, err = kafka.BytesFromStrFieldAsInt32(rmoiFieldFetchMaxPartitionBytes, conf); err != nil {
-				return nil, err
-			}
+			// Configure `start_from_oldest: true`
+			// This is probably not necessary since `__consumer_offsets` is a compacted topic
```

Comment:
I don't really follow this comment.

Reply:
I'm actually not sure... My thought was that `kgo.ConsumeResetOffset(kgo.NewOffset().AtStart())` doesn't do anything when used with a compacted topic, but maybe that's just a silly assumption since I don't really know such topics are read. I removed the comment and left the code as is. Don't think it breaks anything.

Reply:
Compacted topics are not really special from a reader prospective, except they can have gaps in the offsets, because the retention is per key instead of just prefix truncating the log. AtStart will still start from the beginning of the log, but all the old keys might be truncated away. I think it's the right thing to do.